config/kubernetes/org.yaml

```diff
@@ -362,7 +362,6 @@ members:
 - geojaz
 - gerred
 - ggriffiths
-- gianarb
```

Comment:
You don't want to remove yourself here, as this will remove you from the Kubernetes GitHub org altogether.
...unless you also want to do that? 😿

Reply:
I am not sure about how it works, technically I don't have any reason to stay in right now, I a not a member of any other team :) so, I am happy to be part of it if it does not cause any issue ❤️

Reply:
(Resolved via DM.)